packages/babel-plugin-transform-flow-strip-types/src/index.js

```diff
@@ -57,7 +57,16 @@ export default function ({ types: t }) {
           node = node.expression;
         } while (t.isTypeCastExpression(node));
         path.replaceWith(node);
-      }
+      },
+
+      ImportSpecifier(path) {
```

Comment:
I can do that.

FWIW: I wasn't able to actually find this virtual-types file while trying to find where `import type` was stripped (that file seems to be the magical file). I wonder if splitting these up like this is more confusing than it's worth?

(I'll defer to whichever you prefer)